modules/gui.py

```diff
@@ -173,6 +231,28 @@ def browse_source_folder(self):
             self.source_folder_entry.delete(0, tk.END)
             self.source_folder_entry.insert(0, folder_selected)
 
+    def browse_output_folder(self):
+        """Helper function for the browse button of the bag output folder field"""
+        folder_selected = filedialog.askdirectory()
+        if folder_selected:
+            self.output_folder_entry.delete(0, tk.END)
+            self.output_folder_entry.insert(0, folder_selected)
+
+    def set_default_output_filder(self):
+       """Helper function to set output folder as default ('TTS/Saves/Saved Objects' folder)"""
+       # TODO rework USERPROFILE to support *unix
+       output_folder = os.path.join(
+           os.environ["USERPROFILE"],
+           "Documents",
+           "My Games",
+           "Tabletop Simulator",
+           "Saves",
+           "Saved Objects"
+       )
```

Comment:
```suggestion
       if sys.platform == "darwin": # macOS
           base_folder = os.path.join(
           os.path.expanduser("~"),
           "Library",
           )
       else: # windows
           base_folder = os.path.join(
               os.environ["USERPROFILE"],
               "Documents",
               "My Games",
            )
       output_folder = os.path.join(
           f"{base_folder}",
           "Tabletop Simulator",
           "Saves",
           "Saved Objects",
       )
```
This should work (but also need `import sys` at the top)

Reply:
Added your changes, added Linux block